node-webservice/react-client/src/Components/GameplayPage.jsx

```diff
@@ -159,42 +164,55 @@ class GameplayPage extends Component {
       // rotate the part and set 'partHere' to true.
       var partHere = false;
       var copy = [...this.state.parts];
-      copy.forEach(element => {
+      copy.forEach((element,i) => {
+      
         if (element.x === x && element.y === y) {
+          if(this.state.currentType == "empty" && !(x==2 && y==2)){
```

Comment:
=== may be better here instead of ==, there are compiler warnings suggesting to switch it